addons/compat_gm/initSettings.sqf

```diff
@@ -0,0 +1,4 @@
+// primitive way to add to cba setting
```

Comment:
Are we sure this is guaranteed to work?

Reply:
Could fail since this runs at preInit and settings are refreshed one frame after postInit.

If we want to hardcode the item class name, could just add it directly to the setting value in `repair` component. Another solution would be to improve `repair` component to check for `TYPE_TOOLKIT` and support all toolkit type items.

Reply:
added to repair items in 2b3a34807195122c43ea6612eae91ab0c91d529b